docs/design/kata-guest-image-management-design.md

```diff
@@ -49,13 +49,57 @@ Pull the container image directly from the guest VM using `nydus snapshotter` ba
 #### Architecture
 
 The following diagram provides an overview of the architecture for pulling image in the guest with key components. 
-![guest-image-management-architecture](arch-images/guest-image-management-architecture.png)
+```mermaid
+flowchart LR
+    Kubelet[kubelet]--> |1 Pull image request & metadata|Containerd
+    Containerd-->Snapshotter[Nydus Snapshotter]
```

Comment:
Do we want to keep the `[Container Images Registry]-->|2. Pull image metadata| Containerd` to reflect that there is still a host side pull that is needed?

Reply:
The render of `mermaid` would meet some trouble when I used `.`. Let me try to resolve this.